config/config.exs

```diff
@@ -153,6 +153,15 @@ config :logflare, Logflare.Cluster.Utils, min_cluster_size: 1
 config :logflare, Logflare.Alerting.AlertsScheduler,
   init_task: {Logflare.Alerting, :init_alert_jobs, []}
 
+config :logflare, Logflare.Scheduler,
+  jobs: [
+    source_cleanup: [
+      schedule: "*/15 * * * *",
+      task: {Logflare.Sources, :shutdown_idle_sources, []},
+      run_strategy: Quantum.RunStrategy.All
```

Comment:
I don't know how Quantum works, so this question is mainly for my understanding: When configuring Quantum like this, does each node try to schedule a job on all nodes, or does it somehow ensure that only one node actually tries to schedule those jobs? I'd assume that if every node runs the scheduler, `Quantum.Strategy.Local` would be the strategy to choose, since each node would be responsible for scheduling only its own jobs.

Reply:
Yes, thanks for pointing it out, i switched to Quantum.Strategy.Local. using `{Quantum.RunStrategy.All, :cluster}` would result in job execution duplication (seems like i had misconfigured it too which might have caused it to slip through)